Modules/xSharePoint/DSCResources/MSFT_xSPPasswordChangeSettings/MSFT_xSPPasswordChangeSettings.psm1

```diff
@@ -0,0 +1,85 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)][System.String] $MailAddress,
+        [parameter(Mandatory = $false)][ValidateRange(0,356)][System.UInt32] $DaysBeforeExpiry,
+        [parameter(Mandatory = $false)][ValidateRange(0,36000)][System.UInt32] $PasswordChangeWaitTimeSeconds,
+        [parameter(Mandatory = $false)][ValidateRange(0,99)][System.UInt32] $NumberOfRetries
+    )
+
+    Write-Verbose -Message "Retrieving farm wide automatic password change settings"
+
+    $result = Invoke-xSharePointCommand -Arguments $PSBoundParameters -ScriptBlock {
+        $params = $args[0]
+        
+        $farm = Get-SPFarm
```

Comment:
Can you add -ErrorAction Continue to the Get-SPFarm command to ensure the exception doesn't halt the method and the $null value below is returned correctly